src/d3a/models/state.py

```diff
@@ -234,3 +234,12 @@ def market_cycle(self, past_time_slot, time_slot):
         self.charge_history[time_slot] = 100.0 * self.used_storage / self.capacity
         self.charge_history_kWh[time_slot] = self.used_storage
         self.offered_history[time_slot] = self.offered_sell_kWh[time_slot]
+
```

Comment:
One last comment on the PR, the MarketClearingState class could be moved on the market_structures module, in order to be at the same place as Offer, Bid, Trade etc.